scripts/psiphon_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+PSIPHON_PATH=$HOME
+PSIPHON_PYCLIENT_PATH=$PSIPHON_PATH/psiphon-circumvention-system/pyclient
+
+sudo apt-get install mercurial
+cd $PSIPHON_PATH
+hg clone https://bitbucket.org/psiphon/psiphon-circumvention-system
+# optional, compile their ssh
+#cd psiphon-circumvention-system/Server/3rdParty/openssh-5.9p1/
+#./configure
+#make
+#cp ssh ../../../pyclient/
+#cd $PSIPHON_PATH
+
+# create psi_client.dat
+cat <<EOF > psi_generate_dat.py
+#!/usr/bin/env python
+
+import wget
```

Comment:
wget was not installed by default on my system (MacOS 10.10.5 with homebrew) and I needed to `pip install` it.